server/routes/createGoal/reviewView.ts

```diff
@@ -2,15 +2,18 @@ import type { PrisonerSummary } from 'viewModels'
 import type { CreateGoalDto } from 'dto'
 
 export default class ReviewView {
-  constructor(private readonly prisonerSummary: PrisonerSummary, private readonly createGoalDto: CreateGoalDto) {}
+  constructor(
+    private readonly prisonerSummary: PrisonerSummary,
+    private readonly createGoalDto: Array<CreateGoalDto>,
```

Comment:
Very minor, but this should be `createGoalDtos` (ie. plural) - dont worry for now, fix it in your next PR 👍